daliuge-engine/dlg/apps/pyfunc.py

```diff
@@ -30,7 +30,7 @@
 import logging
 import pickle
 
-from typing import Callable
+from typing import Callable, Optional
```

Comment:
Is this used anywhere?

Reply:
Looks like it got taken out, i'll remove all the redundant imports